src/ai/backend/manager/models/acl.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Mapping, Sequence
+
+import graphene
+
+from ai.backend.common.types import VFolderHostPermission
+
+if TYPE_CHECKING:
+    from .gql import GraphQueryContext
+
+
+__all__: Sequence[str] = (
+    "Permission",
+    "get_all_permission",
+)
+
+
+def get_all_permission() -> Mapping[str, Any]:
+    return {
+        "vfolder_permission_list": [perm.value for perm in VFolderHostPermission],
```

Comment:
This should be vFolder "host", not just vFolder.
```suggestion
        "vfolder_host_permission_list": [perm.value for perm in VFolderHostPermission],
```